c4p2n/api.py

```diff
@@ -12,7 +12,11 @@
 from c4p2n.notion import Notion
 
 app = FastAPI()
-notion = Notion(token=config.NOTION_TOKEN, db_link=config.NOTION_LINK)
+notion = Notion(
+    token=config.NOTION_TOKEN,
```

Comment:
Should we consider using a secret field for a Notion token as well?

Reply:
Well, better fix this in another PR.

Reply:
👍